nixos/modules/services/web-apps/pretix.nix

```diff
@@ -63,7 +63,7 @@ in
   };
 
   options.services.pretix = {
-    enable = mkEnableOption "Pretix, a ticket shop application for conferences, festivals, concerts, etc.";
+    enable = mkEnableOption "Pretix, a ticket shop application for conferences, festivals, concerts, etc";
```

Comment:
That could be correct because the first dot belongs to etc not sure if you omit it anyway

Reply:
You never double up periods, so this change is fine.